tests/integration_tests/test_cma.py

```diff
@@ -36,7 +35,7 @@ def test_init_cma_opts() -> None:
             cma_stds={"x": 1, "y": 1},
             seed=1,
             cma_opts={"popsize": 5},
-            independent_sampler=DeterministicRelativeSampler({}, {}),
+            independent_sampler=optuna.samplers.RandomSampler(),
```

Comment:
Not sure but can we simplify by removing the argument altogether? (some ditto occurrences further down)
```suggestion
```

Reply:
I think there are several other places in the diff of this PR where this comment applies. Can we simplify them too?

Reply:
- https://github.com/optuna/optuna/pull/3827/files#diff-59566bd9069c996def7505be23675a39fe165c302f2ddf6c2963dfc1281c7f1a
- https://github.com/optuna/optuna/pull/3827/files#diff-0bc6d8303a01e2fed31f1c1c6cd56fe355216944ce530c5a4fb121fc101f274a
- https://github.com/optuna/optuna/pull/3827/files#diff-0bc6d8303a01e2fed31f1c1c6cd56fe355216944ce530c5a4fb121fc101f274a 

Did you exclude those on purpose?

Reply:
Yes, I deliberately excluded them. In the following lines, mock objects are inserted into `independent_sampler` to check the `call_count`.

Reply:
Ah I see, thanks. Let me resolve this thread.
(For a moment thought about having `independent_sampler` (and other arguments) a public attribute of e.g. `PyCmaSampler` but that's an entirely out-of-scope topic)